src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -1,6 +1,9 @@
 (ns puppetlabs.trapperkeeper.services.metrics.metrics-core
-  (:import (com.codahale.metrics JmxReporter MetricRegistry))
+  (:import (com.codahale.metrics JmxReporter MetricRegistry)
+           com.fasterxml.jackson.core.JsonParseException)
```

Comment:
I think it just imports the `JsonParseException` class, and is functionally the same as what you did above. I'm not familiar with the implementation though, so I may be wrong on that.